### ISSUE TITLE
Autocompletion does no longer work with v2

### ISSUE THREAD

**Issue by sveneberth** [viur-framework/viur-cli#157]
```bash
> eval "$(_VIUR_COMPLETE=bash_source viur)"
bash: Syntaxfehler beim unerwarteten Symbol »(«
> echo "$(_VIUR_COMPLETE=bash_source viur)"
ERROR: The configuration in project.json contains invalid JSON: Expecting value: line 1 column 1 (char 0). Please verify right syntax.
```
I'm using bash.

The `viur-cli` must not require a `project.json` when printing the help.

As workaround I've added a `project.json` into my home dir -- but that's not nice.

**Comment by Grashalmbeisser** [comment]
Should be fixed with #175 . Please Remember to delete the workaround home dir project.json file, since the searh for the file is done recursively.

**Comment by sveneberth** [comment]
> Should be fixed with #175 . Please Remember to delete the workaround home dir project.json file, since the searh for the file is done recursively.

Okay, thanks. I'll wait for the release to test it :)
